system-tests/project-fixtures/svelte/src/lib/Counter.svelte

```diff
@@ -1,10 +1,10 @@
-<script>
-  let count = 0
+<script lang="ts">
```

Comment:
most of the changes to these components are to follow the [svelte migration guide](https://svelte.dev/docs/svelte/v5-migration-guide)

biggest differences:

* similar to React now, is that Svelte components need to explicitly declare `state` or `props` for a given property. This is why there is `Counter.svelte` which uses state and `CounterProp.svelte` which uses a prop to set the counter.
*  Events are NOT propagated up to components by default and event dispatchers are deprecated. The recommended pattern is now to pass a callback function in as a prop and call that when the event is fired (see [Changes to Component Events](https://svelte.dev/docs/svelte/v5-migration-guide#Event-changes-Component-events)) 

There are a few others but these are the main changes